.github/workflows/build_docker_image_wheelbuilder_linux.yml

```diff
@@ -0,0 +1,26 @@
+
+name: Build docker image wheelbuilder-linux
+
+on:
+  push:
+    branches:
+      - 'master'
```

Comment:
```suggestion
  pull_request:
```

Reply:
Are you sure? With trigger on `pull_request:` image will be built and published when pull request is created, not merged

Reply:
Thats what I am doing in the other image-actions
My thinking was that if I want to "debug" the creation of the image, then if I only trigger it when merging to master then I need one PR per image build which seems cumbersome

What do you think?

Reply:
for example this PR has not triggered the builds yet, so how do you know they will suceed?

Reply:
I've been debugging actions on a fork, but I see your point. On the other hand, there is a risk, that a defective image would get pushed do dockerHub, whithout any code review

Reply:
hm that is a good point...
I guess you are right, I don't have a strong opinion

After all debugging the DockerFile in Github is probably not best practice :sweat_smile: (I can only hope none is looking through my commit history :smile:)

Then lets agree on this and I will update the other workflow files, since I am anyway working a bit on them